backend/core/src/shared/email.service.ts

```diff
@@ -46,6 +86,33 @@ export class EmailService {
     )
   }
 
+  private partial(view: string) {
+    return fs.readFileSync(
+      path.join(path.resolve(__dirname, "..", "..", "views").replace("/dist", ""), `/${view}`),
```

Comment:
Is this meant for production builds when we're inside of a `dist` folder? Might it be cleaner to make sure that the build process copies over `views` instead?

Reply:
that would be better, true. I'm not sure how to do that? Can you send me an example @bencpeters ?

Reply:
Ya, I did this in my authorization PR right [here](https://github.com/bloom-housing/bloom/blob/backend-authorization-and-roles/backend/core/nest-cli.json#L13). Basically you tell the Nest build system to copy the files over. The only subtlety is that the way our build system is set up right now, the TS files get compiled to `dist/src/path/to/file.js`, whereas when you specify an entry in `assets` it goes from `src/path/to/file` to `dist/path/to/file`, and what we want with `__dirname`-based relative paths is `dist/src/path/to/file`, hence the "custom" definition with `outDir`.